### PR TITLE
Update playground and SDK to use recent SPAJS

### PR DESCRIPTION
**Pull request by frederikprijck**

### CODE REVIEW

File: src/auth0-provider.tsx [hunk 1]
```diff
@@ -291,7 +291,7 @@ const Auth0Provider = (opts: Auth0ProviderOptions): JSX.Element => {
         throw tokenError(error);
       }
       const user = await client.getUser();
-      if (user.updated_at !== userUpdatedAt) {
+      if (user?.updated_at !== userUpdatedAt) {
```
Comment (frederikprijck) [inline src/auth0-provider.tsx:294]: Looks like this has changed in SPAJS that breaks current consumers. Need to discuss if this is fine or if we should rollback SPAJS.

File: static/index.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
       crossorigin
       src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"
     ></script>
-    <script src="https://cdn.auth0.com/js/auth0-spa-js/1.7/auth0-spa-js.production.js"></script>
+    <script src="https://cdn.auth0.com/js/auth0-spa-js/1.13.5/auth0-spa-js.production.js"></script>
```
Comment (adamjmcgrath) [inline static/index.html:24]: This shouldn't actually be there, `auth0-react.js` on the line below will bundle spa js - can you remove it

Comment (frederikprijck) [reply static/index.html]: Updated

**Review by adamjmcgrath — APPROVED** [review]

File: src/auth0-provider.tsx (hunk 1 above)
Comment (twbryan) [reply src/auth0-provider.tsx:294]: I was running into this bug (TypeError: Cannot read property 'updated_at' of undefined), so glad this change was committed.

**Pull request merged**